_data/meltano/extractors/tap-mysql/transferwise.yml

```diff
@@ -19,38 +19,66 @@ settings:
 - name: host
   value: localhost
   label: Host
-  description: The MySQL hostname.
+  description: The MySQL/MariaDB hostname.
 - name: port
   label: Port
   kind: integer
   value: 3306
-  description: The MySQL port.
+  description: The MySQL/MariaDB port.
 - name: user
   label: User
-  description: The MySQL username.
+  description: The MySQL/MariaDB username.
 - name: password
   kind: password
   label: Password
-  description: The MySQL password.
+  description: The MySQL/MariaDB password.
 - name: database
   label: Database
-  description: The MySQL database name.
+  description: The MySQL/MariaDB database name.
+- name: cursorclass
+  label: Cursor Class
+  value: pymysql.cursors.SSCursor
+  description: Cursor class used by PyMYSQL.
+- name: server_id
+  label: Server ID
+  kind: integer
+  value: <random int>
```

Comment:
In order to let the tap derive the random ID here, I think we need to remove this line. Otherwise, I think '<random int>' will fail conversion to type 'int'.

```suggestion
```